Organization/Organization/MainViewController.m

```diff
@@ -1,57 +1,67 @@
 #import "MainViewController.h"
 #import "DetailViewController.h"
 #import "CreateEmployeeViewController.h"
-#import "Employee.h"
-#import "Organization.h"
+#import "Employee+CoreDataClass.h"
+#import "Organization+CoreDataClass.h"
+#import "AppDelegate.h"
 
-@interface MainViewController ()		
 
-@end
 
 @implementation MainViewController
 
+
+
+NSString* attributeValue = @"Siemens";
```

Comment:
Local constants should be declared as `static NSString *const AttributeValue = @"Siemens";` and outside of class (after imports)